### PR TITLE
test(accordion): adds unit tests

### LINKED ISSUE #568: Add unit tests to the Accordian components
Add unit tests for the accordian and accordian-toolbar components

### PR DESCRIPTION
**Pull request by andrew-frueh**
adds unit tests for component and adds two-way binding to open property

closes #568

### CODE REVIEW

File: projects/cashmere/src/lib/accordion/accordion.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    triggerVal: boolean = true;
+    openVal: boolean = false;
+
+    onOpenStart: (event?: Event) => void = () => {};
+    onOpen: (event?: Event) => void = () => {};
+    onCloseStart: (event?: Event) => void = () => {};
+    onClosed: (event?: Event) => void = () => {};
+}
+
+fdescribe('AccordionComponent', () => {
```
Comment (joeskeen) [inline projects/cashmere/src/lib/accordion/accordion.component.spec.ts]: Remove f

File: projects/cashmere/src/lib/accordion/accordion.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    it('should emit open events when the accordion open animation is complete', () => {
+        let accordionComponent = fixture.debugElement.query(By.directive(AccordionComponent));
+        spyOn(component, 'onOpen');
+        spyOn(component, 'onOpenStart');
+
+        accordionComponent.componentInstance.toggleOpen();
+        fixture.detectChanges();
+
+        fixture.whenStable().then(() => {
```
Comment (joeskeen) [inline projects/cashmere/src/lib/accordion/accordion.component.spec.ts]: Return at the start of this line

File: projects/cashmere/src/lib/accordion/accordion.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        let accordionComponent = fixture.debugElement.query(By.directive(AccordionComponent));
+        spyOn(component, 'onClosed');
+        spyOn(component, 'onCloseStart');
+
+        component.openVal = true;
+        fixture.detectChanges();
+        accordionComponent.componentInstance.toggleClose();
+        fixture.detectChanges();
+
+        fixture.whenStable().then(() => {
```
Comment (joeskeen) [inline projects/cashmere/src/lib/accordion/accordion.component.spec.ts]: Return

File: projects/cashmere/src/lib/accordion/accordion.component.spec.ts (hunk 1 above)
Comment (andrew-frueh) [reply projects/cashmere/src/lib/accordion/accordion.component.spec.ts]: gah - thanks Joe.  I do that every time :-)

File: projects/cashmere/src/lib/accordion/accordion.component.spec.ts (hunk 2 above)
Comment (andrew-frueh) [reply projects/cashmere/src/lib/accordion/accordion.component.spec.ts]: Got it, so by adding return it ensures `whenStable` won't throw off the async of other tests?

**Pull request merged**